object/error.go

```diff
@@ -1,19 +1,23 @@
 package object
 
+// SplitInfoError is a special errors to show the original object was split to parts.
```

Comment:
`is a special error that means that the original object is a large one (split into a number of smaller objects)`?

We call them "large" in the spec.

Btw, why it's not an `ErrSplitInfo`?

Reply:
Your variant is more laconic. It would `ErrSplitInfo` in case of a simple error, without a content.
```go
var (
ErrSplitInfo = errors.New("error text")
)
```
Our SplitInfoError contains SplitInfo, and this is a different format for errors structs